### ISSUE TITLE
Named colors

### ISSUE THREAD

**Issue by JamieBallingall** [JordanMartinez/purescript-halogen-svg-elems#2]
When I was porting the [Elm Shapes Example](https://elm-lang.org/examples/shapes) I saw code such as:
```
[ circle
    [ cx "50"
    , cy "50"
    , r "40"
    , fill "red"
    , stroke "black"
    , strokeWidth "3"
    ]
```
I had to replace the color names "red" and "black" with direct RGB codes:
```
[ circle
    [ cx 50.0
    , cy 50.0
    , r 40.0
    , fill $ Just $ RGB 255 0 0
    , stroke $ Just $ RGB 0 0 0
    ]
```
Personally, I never use named colors but it would be nice to have them, if only for introductory examples. I'm happy to make a PR for this. Unless anyone has any serious objections?

**Comment by antarestrader** [comment]
I think it would be appropriate to add a String constructor to the [Color type](https://github.com/JordanMartinez/purescript-halogen-svg-elems/blob/4be7f01aefcddb76f89dbab3d02d8cccfba5458f/src/Halogen/Svg/Attributes.purs#L84-L85) and the accompanying printColor function

**Comment by JamieBallingall** [comment]
This was resolved with 0b281a71dbb7900437379e8d9dff70d742a246ee. Closing.
